src/lib/pickles/plonk_checks/scalars.ml

```diff
@@ -270,18 +654,977 @@ module Tick : S = struct
+                                             "0x0000000000000000000000000000000000000000000000000000000000000001"
+                                           + beta )
+                                       , fun () ->
+                                           field
+                                             "0x0000000000000000000000000000000000000000000000000000000000000001"
+                                       ) ) )
+                             + if_feature
+                                 ( LookupPattern Xor
+                                 , (fun () ->
+                                     ( field
+                                         "0x0000000000000000000000000000000000000000000000000000000000000001"
+                                     + beta )
+                                     * if_feature
+                                         ( LookupsPerRow 2
+                                         , (fun () ->
+                                             field
+                                               "0x0000000000000000000000000000000000000000000000000000000000000001"
+                                             + beta )
+                                         , fun () ->
+                                             field
+                                               "0x0000000000000000000000000000000000000000000000000000000000000001"
+                                         )
+                                     * if_feature
+                                         ( LookupsPerRow 3
+                                         , (fun () ->
+                                             field
+                                               "0x0000000000000000000000000000000000000000000000000000000000000001"
+                                             + beta )
+                                         , fun () ->
+                                             field
+                                               "0x0000000000000000000000000000000000000000000000000000000000000001"
+                                         )
+                                     * if_feature
+                                         ( LookupsPerRow 4
+                                         , (fun () ->
+                                             field
+                                               "0x0000000000000000000000000000000000000000000000000000000000000001"
+                                             + beta )
+                                         , fun () ->
+                                             field
+                                               "0x0000000000000000000000000000000000000000000000000000000000000001"
+                                         )
+                                     * ( gamma
+                                       + ( joint_combiner
+                                           * ( joint_combiner
+                                               * cell (var (Witness 11, Curr))
```

Comment:
So this refers to the column positions of the first Xor made (3 xor 7 = 11), then the second (4 xor 8 = 12), blah, until the 4th xor. Okay. I checked them all, seem to have the right indices.